firmware/src/core/configmanager/ConfigManager.h

```diff
@@ -0,0 +1,101 @@
+#ifndef CONFIG_MANAGER_H
+#define CONFIG_MANAGER_H
+
+#include "OrbsWiFiManager.h"
+#include "WifiManagerCustomParameters.h"
+#include <Preferences.h>
+#include <functional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#define CM_PARAM_TYPE_STRING 0
```

Comment:
an enum would be a better way of doing this

Reply:
Done